### ISSUE TITLE
 Images in figure 2 are not aligned

### ISSUE THREAD

**Issue by rparkr** [arXiv/html_feedback#2118]
### Description

The images in figure 2 of https://arxiv.org/html/2408.00653v1 are shifted downward when viewed in a Google Pixel 4a device using Firefox for Android.

### (Optional:) Please add any files, screenshots, or other information here.

![figure 2](https://github.com/user-attachments/assets/9dcad902-17e9-4140-889f-4e6be206c654)

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

fbfa5d12-a18e-4539-9836-32b125d681b8

### Paper URL

https://arxiv.org/html/2408.00653v1

### Browser

Firefox/130.0

### Device Type

Android

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

This is not yet the perfect possible rendering, but it is a significant improvement from the screenshot - there will be future improvements as the collection gets reconverted with the latest latexml.

**Comment by rparkr** [comment]
Yes, the new rendering looks great now! Thanks for your efforts on this, arXiv HTML formats have been excellent for quickly reviewing on my mobile device (compared to PDFs).
